authors/apps/authentication/utils.py

```diff
@@ -6,28 +6,30 @@
 from django.template.loader import render_to_string
 
 # Method to Send emails
-def send_link(email):
+
+
+def send_link(email, subject, template, url):
     payload = {'email':  email,
-                "iat": datetime.now(),
-                "exp": datetime.utcnow()
-                + timedelta(minutes=5)}
+               "iat": datetime.now(),
+               "exp": datetime.utcnow()
+               + timedelta(minutes=5)}
     token = jwt.encode(payload,
-                        settings.SECRET_KEY,
-                        algorithm='HS256').decode('utf-8')
+                       settings.SECRET_KEY,
+                       algorithm='HS256').decode('utf-8')
     from_email, to_email = settings.EMAIL_HOST_USER, [email]
-    subject = "Authors Haven Verification Link"
-    site_url = os.getenv('EMAIL_VERIFICATION_URL')
+    site_url = os.getenv('APP_BASE_URL')
     link_url = str(site_url) + \
-        '/api/users/verify/{}/'.format(token)
-
+        url + '{}/'.format(token) 
```

Comment:
I like how you've edited this function to serve your new purpose but I guess we won't need to send a token when we want to share an article by email. How about we pass in a token, from the view, as an argument if we need to include token. If not, we send email without a token.